Drivers/Src/Minions.c

```diff
@@ -9,8 +9,8 @@ static OS_MUTEX OutputMutex; //Mutex to lock GPIO writes to input pins
 
 //should be in sync with pin enum
 const PinInfo_t PINS_LOOKARR[MINIONPIN_NUM] = {
-    {GPIO_Pin_0, PORTA, INPUT},
     {GPIO_Pin_1, PORTA, INPUT},
+    {GPIO_Pin_0, PORTA, INPUT},
```

Comment:
Do we need to change the enum to match this?

Reply:
Yes! I will change that. Also in reference to mirroring enums, I believe that the CANId_t enums do not completely mirror the CAN LUT. Should I also update the CAN LUT in this issue ticket?

Reply:
Also, NUM_CAN_IDS is currently 1410, which is the incremented number following CARDATA. Is that intended or should it be the number of CAN IDs as enums (21 IDs)?

Reply:
It's intended to be a big array, since it makes looking up CAN IDs quicker

Reply:
Oh you're right. I completely forgot about that.
Intended behavior is that each CAN ID acts as an index to the CAN LUT array, so it makes sense that the CAN LUT size is one more than the maximum possible CAN ID. This way we can just look up the CAN spec we want using the CAN ID as an index, as opposed to making it an array of structs each with its own CAN ID (in which case we'd have to linear search for the CAN ID). However, proper naming might be good here. How about MAX_CAN_ID?